src/message.rs

```diff
@@ -229,6 +229,12 @@ impl<const N: usize> From<[u8; N]> for Message {
     }
 }
 
+impl From<u8> for Message {
```

Comment:
Is this impl really useful? Conceptually it doesn't make a lot of sense...

Reply:
The usecase I personally found it sending a status response, i.e. a boolean, back to the other client. In which case it currently looks roughly like:
```rust
[struct.success as u8].as_slice()
```
Which would then be:
```rust
struct.success as u8
```

Initially I wanted to implement the `From<T> for Message` for anything that's either `Into<u8>` or `AsRef<[u8]>`, but that currently does not work without removing the `impl`s for `Box` and `Vec`.

Reply:
With your other impl it's just `[struct.success as u8]` which seems good enough.